src/main/java/seedu/address/logic/commands/AddIntervieweeCommand.java

```diff
@@ -19,7 +19,7 @@
 
     public static final String COMMAND_WORD = "interviewee";
     public static final String MESSAGE_SUCCESS = "New interviewee added: %1$s";
-    public static final String MESSAGE_FORMAT = "add " + COMMAND_WORD + " <full name> [-aka <alias>]";
```

Comment:
I think this needs to be `<full name>` still (since this is when we give the full name of the interviewee!)

Only when we are referring to the interviewee object in the app, then we use <interviewee> which means they can write the full name, alias or id. It would be weird if when adding a new interviewee they give the alias